src/PKSim.CLI.Core/Services/QualificationRunner.cs

```diff
@@ -104,7 +104,10 @@ public async Task RunBatchAsync(QualificationRunOptions runOptions)
             //We run the output, this is for the old matlab implementation where we need xml. Otherwise, we only need pkml export
             ExportMode = runOptions.Run ? SimulationExportMode.Xml | SimulationExportMode.Csv : SimulationExportMode.Pkml,
 
-            Simulations = config.Simulations
+            Simulations = config.Simulations,
+
+            //We only want to export what is required in this case
+            ExportAllSimulationsIfListIsEmpty = false
```

Comment:
In the qualificationrunner, we set it to false because well...this is what we need:)